projects/angular/src/data/datagrid/providers/items.ts

```diff
@@ -13,6 +13,8 @@ import { FiltersProvider } from './filters';
 import { Page } from './page';
 import { Sort } from './sort';
 
+export type ClrDatagridItemsTrackByFunction<T> = (item: T) => any;
```

Comment:
Is there a reason we need the new type here? Are we deviating from the current type so the docs get updated?

Reply:
The Angular `TrackByFunction<T>` take the index as the first parameter. I removed that parameter because the index of the item isn't available when `clrDgSelectable` is set because `items.all` isn't set until `ClrDatagrid#ngAfterContentInit`.